pom.xml

```diff
@@ -101,6 +101,12 @@
 				</configuration>
 			</plugin>
 
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-failsafe-plugin</artifactId>
+				<version>3.0.0-M8</version>
```

Comment:
latest version is 3.0.0-M9: https://search.maven.org/artifact/org.apache.maven.plugins/maven-failsafe-plugin